Manifest.toml

```diff
@@ -4,16 +4,16 @@
 uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 
 [[BinaryProvider]]
-deps = ["Libdl", "Pkg", "SHA", "Test"]
```

Comment:
We actually don't need a manifest at all, do we? Maybe simply delete this file?

Reply:
I would definitely go with that and delete it. I'm not honestly sure in what circumstances it's useful to have a manifest file in a package - I can see the value in a project, but not in a package, but maybe I' m missing something?

Reply:
I agree

Reply:
There are different opinions among core devs - I think Stefan likes including it because it's an environment where tests are guaranteed to work, and if you're just adding the package, it doesn't hurt.

That said, I usually don't include it in my packages. I just left it in because it was already there, happy to remove it.